spec/BBCoreSpec.js

```diff
@@ -867,8 +867,10 @@ describe("BBCore.videoRecorder", function() {
         setupMockApiRequest(result.withOptionsSuccess);
 
         bbCore.getVideoRecorder(opts, successCallbackSpy);
-
-        expect($.ajax.calls.argsFor(0)[0].data).toEqual(opts);
```

Comment:
The reason this test was passing before and hiding the bug is because inside of `getVideoRecorder` `$.extend()` was mutating `opts`.